litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/TableData.tsx

```diff
@@ -194,25 +178,14 @@ const TableData: React.FC<TableDataProps> = ({ data, deleteRow }) => {
                 : ''
             }
           >
-            {isOpen ? (
-              <div className={classes.expDiv}>
-                <Typography className={classes.expInfoActive}>
-                  <strong>
-                    {t('chaosWorkflows.browseSchedule.showExperiment')}
-                  </strong>
-                </Typography>
-                <KeyboardArrowDownIcon className={classes.expInfoActiveIcon} />
-              </div>
-            ) : (
-              <div className={classes.expDiv}>
-                <Typography className={classes.expInfo}>
-                  <strong>
-                    {t('chaosWorkflows.browseSchedule.showExperiment')}
-                  </strong>
-                </Typography>
-                <ChevronRightIcon />
-              </div>
-            )}
+            <div className={classes.expDiv}>
+              <Typography className={classes.expInfo}>
+                <strong>
+                  {t('chaosWorkflows.browseSchedule.showExperiment')}
+                </strong>
```

Comment:
can we use a fontWeight style in the css class above rather than this extra `<strong>` jsx.